config/initializers/two_factor_auth.rb

```diff
@@ -0,0 +1,4 @@
+
+Rails.application.configure do
+  config.use_2fa = !Rails.env.development? || ENV['FORCE_2FA'].present?
```

Comment:
Maybe we can use something like

`config.use_2fa = (Rails.env.development? && ENV['FORCE_2FA'].present?) || !ENV['DISABLE_2FA']`